ra2ce/ra2ce_handler.py

```diff
@@ -48,8 +53,60 @@ class Ra2ceHandler:
     input_config: ConfigWrapper
 
     def __init__(self, network: Optional[Path], analysis: Optional[Path]) -> None:
-        self._initialize_logger(network, analysis)
-        self.input_config = ConfigFactory.get_config_wrapper(network, analysis)
+        if network or analysis:
+            self._initialize_logger(network, analysis)
+            self.input_config = ConfigFactory.get_config_wrapper(network, analysis)
+
+    @classmethod
+    def from_config(
+        cls, network: NetworkConfigData, analysis: AnalysisConfigData
+    ) -> Ra2ceHandler:
+
+        def _set_config_paths(_analysis_config: AnalysisConfigWrapper) -> None:
```

Comment:
No need to have these methods as private, they are already internal methods.

Reply:
Adjusted